### DISCUSSION TITLE
Calendar view settings - Start of visible time range/ End of visible time range

### DISCUSSION THREAD

**Discussion by martin-chalupa** [kimai/kimai#5016] [General]
Hello, after setting the calendar view to show only hours from 6:00 to 19:00, I can still input activity outside of this time range. This causes no problems on desktop, but is more difficult to input in cell phone (employee must everytime scroll way down to eligible hours). Is there a way to hide these hours in the acitivity input?
![image](https://github.com/user-attachments/assets/ef25a77c-4ce6-4e95-a0ef-99137755f12b)

**Comment by Rudiberto** [comment]
These settings are working for the calendar sight only, as I understand. You could choose "now" as the start standard for Default Start-time, so that users do not have to choose any start time.
Using "Time-Clock" as Time tracking mode allows neither choosing any time, but only start and stop. A good choice on mobile devices, in my opinion.
